### PR TITLE
Googleplay verification added

### PR DESCRIPTION
**Pull request by thaelathy**
Based on issue
https://github.com/dotpot/InAppPy/issues/1

Made a class that verifies receipt in google play.

Patched Apple validator to set timeout for http requests (default 60 seconds is sometimes too much).

Based on issue
https://github.com/dotpot/InAppPy/issues/2

Modified initializer of Exception base class to accept error message *and* raw_response.

Pls, consider taking a look.

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -5,3 +5,4 @@ envlist = py{27,35}
 commands = py.test -v
 deps =
     pytest
+    -rrequirements.txt
```
Comment (dotpot) [inline tox.ini]: `-r requirements.txt`

**Comment by dotpot** [comment]
also tests for `GooglePlayVerifier` would be nice ;)

File: tox.ini (hunk 1 above)
Comment (thaelathy) [reply tox.ini]: not a typo, but a syntax. https://tox.readthedocs.io/en/latest/example/basic.html#depending-on-requirements-txt-or-defining-constraints

**Comment by thaelathy** [comment]
> also tests for `GooglePlayVerifier` would be nice ;)

Hi! I added some tests

**Comment by dotpot** [comment]
Thanks a lot for contribution! Also you might want to update readme with information about Googleplay verification ;)

**Comment by thaelathy** [comment]
Check out the docs pls

**Comment by dotpot** [comment]
Thanks! I will do a new pip release sometime soon ;)

**Pull request merged**